skrf/media/media.py

```diff
@@ -836,10 +836,12 @@ def line(self, d: NumberLike, unit: str = 'deg',
 
         kwargs.update({'z0':z0})
         s_def = kwargs.pop('s_def', S_DEF_DEFAULT)
-        # Need to use either traveling or pseudo definition here
-        # for the network to match traveling waves.
-        # the definition of the line trough a match and a delay would not
-        # works for complex characteristic impedances otherwise.
+        # The use of either traveling or pseudo waves s-parameters definition
+        # is required here.
+        # A line with a complex characteristic impedance equal to port
+        # impedance does not have zero reflection coefficients with the power
+        # waves definition.
+        # Instead, reflection coefficients will have conjugation. 
```

Comment:
This sounds misleading. The definition of the reflection coefficient for powerwaves has conjugation.

Reply:
Just to be sure of my understanding: if we have a line of characteristic impedance 50 + 1j ohm, we should have port impedances of 50 - 1j ohm to have S11 equal to 0, is this correct?

Reply:
Maybe shall we just replace the 4 last comment line by ”The definition of the reflection coefficient for power waves has conjugation." 😅